tests/copp/test_copp.py

```diff
@@ -115,6 +117,8 @@ def test_add_new_trap(self, duthosts, enum_rand_one_per_hwsku_frontend_hostname,
         3. Verify the trap status is installed by sending traffic
         """
         duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+        if duthost.is_multi_asic:
+            pytest.skip("Skipping as test needs changes to supoort multi-asic dut")
```

Comment:
can we add conditional mark iso for skipping here?
Example:
https://github.com/sonic-net/sonic-mgmt/blob/fb6971e883e9d335b5d648d06dd7bbecccf56744/tests/common/plugins/conditional_mark/tests_mark_conditions.yaml#L655-L659